@here/harp-mapview/lib/DecodedTileHelpers.ts

```diff
@@ -457,26 +458,49 @@ export function applyTechniqueToMaterial(
     level?: number,
     skipExtraProps?: string[]
 ) {
-    Object.getOwnPropertyNames(technique).forEach(propertyName => {
+    const commonProps = Object.getOwnPropertyNames(technique).filter(propertyName => {
         if (
             propertyName.startsWith("_") ||
             BASE_TECHNIQUE_NON_MATERIAL_PROPS.indexOf(propertyName) !== -1 ||
             DEFAULT_SKIP_PROPERTIES.indexOf(propertyName) !== -1 ||
             (skipExtraProps !== undefined && skipExtraProps.indexOf(propertyName) !== -1)
         ) {
-            return;
+            return false;
         }
         const prop = propertyName as keyof typeof technique;
         const m = material as any;
         if (typeof m[prop] === "undefined") {
-            return;
+            return false;
         }
+        return true;
+    });
+
+    // Remove transparent color from the firstly processed properties set.
+    let transparentColor = getTechniqueTransparentColor(technique);
```

Comment:
This is actually `transparentColorPropName`.

Reply:
Fixed.